qcodes/instrument_drivers/Keysight/keysight_e4980a.py

```diff
@@ -293,6 +376,14 @@ def _set_measurement(self,
         self._measurement_pair = measurement_pair
         self.write(f":FUNCtion:IMPedance {measurement_pair.name}")
 
+    def _options(self) -> Tuple[str, ...]:
+        """
+        Returns installed options numbers. The options are Power/DC Bias
```

Comment:
right? "possible"? or are these more options?
```suggestion
        Returns installed options numbers. The possible options are Power/DC Bias
```

Reply:
Yes, there are more options possible

Reply:
should then the sentence be reworded so that it's clear that it mentions only two options while there are actually more options?